src/AppFrame.tsx

```diff
@@ -1,28 +1,130 @@
-/* eslint-disable @typescript-eslint/no-floating-promises */
 import React, { useCallback, useState } from "react";
 import { useLocation } from "react-router-dom";
 import { Frame, Layout, Navigation, Page, TopBar } from "@shopify/polaris";
 import { HomeMajor } from "@shopify/polaris-icons";
 import { Routes } from "./Routes";
-import { TwitterTimelineEmbed } from "react-twitter-embed";
 import { useTranslation } from "react-i18next";
 import i18next from "i18next";
+import { ItemProps } from "@shopify/polaris/types/latest/src/components/Navigation/components";
+import { Twitter } from "./components/Twitter";
+
+const languages = [
+  {
+    langCode: "en",
+    label: "English",
+  },
+  {
+    langCode: "fr",
+    label: "Français",
+  },
+  {
+    langCode: "ar",
+    label: "العربية",
+  },
+  {
+    langCode: "bn",
+    label: "বাংলা",
+  },
+  {
+    langCode: "de",
+    label: "Deutsch",
+  },
+  {
+    langCode: "es",
+    label: "Español",
+  },
+  {
+    langCode: "hr",
+    label: "Hrvatski",
+  },
+  {
+    langCode: "it",
+    label: "Italiano",
+  },
+  {
+    langCode: "he",
+    label: "עברית",
+  },
+  {
+    langCode: "hu",
+    label: "Magyar",
+  },
+  {
+    langCode: "ja",
+    label: "日本語",
+  },
+  {
+    langCode: "pl",
+    label: "Polski",
+  },
+  {
+    langCode: "pt",
+    label: "Português",
+  },
+  {
+    langCode: "ro",
+    label: "Română",
+  },
+  {
+    langCode: "ru",
+    label: "Русский",
+  },
+  {
+    langCode: "sr",
+    label: "Srpski",
+  },
+  {
+    langCode: "ta",
+    label: "தமிழ்",
+  },
+  {
+    langCode: "tl",
+    label: "Tagalog",
+  },
+  {
+    langCode: "tr",
+    label: "Türkçe",
+  },
+  {
+    langCode: "zh-CN",
+    label: "简体中文",
+  },
+  {
+    langCode: "zh-TW",
+    label: "繁體中文",
+  },
+];
+
+function getLanguageBarItemProps(): ItemProps[] {
+  const output: ItemProps[] = languages.map((language) => {
+    const selected =
+      i18next.language === language.langCode ||
+      i18next.language?.substring(0, 2) === language.langCode;
+    return {
+      url: "/",
+      label: language.label,
+      onClick: () => {
+        i18next
+          .changeLanguage(language.langCode)
+          .then(() => {
```

Comment:
These are gratuitous. You don't need the `then`, but you may need to the `catch` IIRC because this would be a floating promise otherwise. A `console.error` statement could be added until we add logging - or we could add actual logging with winston or something - but this is fine for the time being.

Reply:
I did this solely to stop eslint from complaining.

Reply:
Yep - but the purpose of the eslint rule is to prevent a possible error from a async fire and forget call to be swallowed, which makes it difficult to track down.